src/TuDa.CIMS.Web/Components/SubmitPopup.razor

```diff
@@ -0,0 +1,54 @@
+@* Submit Popup *@
+<MudDialog>
+    <TitleContent></TitleContent>
+    <DialogContent>
+        <MudContainer>
+            <MudGrid Class="justify-space-between" Style="max-width: 400px;">
+                <MudItem>
+                    <MudText Typo="Typo.h6">Arbeitsgruppe auswählen</MudText>
+                </MudItem>
+                <MudItem Class="d-flex align-items-center" Style="margin-bottom: 300px;">
+                    <MudAutocomplete T="string" Label="Professor" SearchFunc="@Search">
```

Comment:
🔧 `T` should be `WorkingGroup` and a `ToStringFunc` should be provided
For inspiration, look at #99